spanner/api/Spanner.Samples/BatchReadRecordsAsync.cs

```diff
@@ -30,7 +30,7 @@ public class BatchReadRecordsAsyncSample
         using var connection = new SpannerConnection(connectionString);
         await connection.OpenAsync();
 
-        using var transaction = await connection.BeginReadOnlyTransactionAsync();
+        using var transaction = await connection.BeginTransactionAsync(SpannerTransactionCreationOptions.ReadOnly, cancellationToken: default);
```

Comment:
This one needs to be:

```suggestion
        using var transaction = await connection.BeginTransactionAsync(
            SpannerTransactionCreationOptions.ReadOnly.WithIsDetached(true), cancellationToken: default);
```

Reply:
Updated!